c/src/CMakeLists.txt

```diff
@@ -31,10 +31,18 @@ set(libsbp_SRCS
   edc.c
   sbp.c
   )
-set(LIBSBP_CFLAGS "-fpic" CACHE STRING "Compile flags for libsbp.")
```

Comment:
Libsbp is a library that our customers definitely use, we need to make sure we're choices that reflect that.  The cflags var here we designed to allow customers to override these compile flags if needed.

Reply:
Ok, the old use wasn't working when I added more than one flag to the string. I'll figure out the right want to do this.

Would the removed CMake files potentially be used by customers?

Reply:
Not sure, can we investigate the `git blame` on these files to see why they were originally introduced?

Reply:
Most of the `.cmake` files appear to have been added in this commit: https://github.com/swift-nav/libsbp/commit/062a3e3ec351303023e556a95ee880b6e11e2479 but they weren't used in that commit, and haven't been touched since they were added. The toolchain file has been updated recently so I'll bring it back in.

Reply:
Ok, SGTM.